ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/static-routes.j2

```diff
@@ -4,17 +4,18 @@
 
 ### Static Routes Summary
 
-| VRF | Destination Prefix | Next Hop IP             | Exit interface      | Administrative Distance       | Tag               | Route Name                    | Metric         |
-| --- | ------------------ | ----------------------- | ------------------- | ----------------------------- | ----------------- | ----------------------------- | -------------- |
+| VRF | Destination Prefix | Next Hop IP | Exit interface | Track Route | Administrative Distance | Tag | Route Name | Metric |
```

Comment:
To avoid cluttering the docs for every (rarely used) knob, we could consider adding ` (tracked with BFD)` to the next-hop IP? I am fine with both.

Reply:
I agree with this suggestion